src/SequenceManager.cc

```diff
@@ -100,10 +100,12 @@ SequenceManager::copy_sequence(UniqueId original_uid, const SequenceName& new_na
 
     const auto old_name = find_sequence_on_disk(original_uid, sequences).path;
 
-    perform_commit({ }, gul14::cat("Copy sequence ", old_name.string(), " to "), [&]()
-        {
-            return write_sequence_to_disk(seq);
+    auto ok = perform_commit(gul14::cat("Copy sequence ", old_name.string(), " to "),
+        [this, &seq]() {
+            return this->write_sequence_to_disk(seq);
         });
+    if (not ok)
+        Error(cat("Cannot commit sequence copy ", to_string(original_uid)));
```

Comment:
That is supposed to be a `throw`, right?

Reply:
:woman_facepalming: Thank you